src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx

```diff
@@ -126,13 +125,13 @@ export const ContentMessageComponent: React.FC<ContentMessageProps> = ({
       return true;
     }
 
-    const currentMessageDate = fromUnixTime(message.timestamp() / TIME_IN_MILLIS.SECOND);
-    const previousMessageDate = fromUnixTime(previousMessage.timestamp() / TIME_IN_MILLIS.SECOND);
+    // Interval in milliseconds, within which messages are grouped together
+    const GROUPED_MESSAGE_INTERVAL = 30000;
```

Comment:
Let's use the `TIME_IN_MILLIS` enum there (to make things easy to read)

```suggestion
    const GROUPED_MESSAGE_INTERVAL = 30 * TIME_IN_MILLIS.SECOND;
```

Reply:
Oh no, magic number warning :( 
Better to keep it here for clarity though in my opinion (instead of `currentMessageDate - previousMessageDate >= GROUPED_MESSAGE_INTERVAL * TIME_IN_MILLIS.SECOND`)

Reply:
nah it's okay, this magic number warning doesn't really make sense here (we need to fine tuned this one, I find it a little stupid sometimes)